mlflow/utils/file_utils.py

```diff
@@ -581,11 +582,13 @@ def get_or_create_tmp_dir():
 
     if is_in_databricks_runtime() and get_repl_id() is not None:
         # Note: For python process attached to databricks notebook, atexit does not work.
-        # The /tmp/repl_tmp_data/{repl_id} directory will be removed once databricks notebook
-        # detaches.
-        # The repl_tmp_data directory is designed to be used by all kinds of applications,
+        # The directory returned by `dbutils.entry_point.getReplLocalTempDir()`
+        # will be removed once databricks notebook detaches.
+        # The temp directory is designed to be used by all kinds of applications,
```

Comment:
```suggestion
        # The temporary directory is designed to be used by all kinds of applications,
```

Not a big issue, I usually avoid shortening words in comments.